jenkins/deploy.sh

```diff
@@ -82,24 +82,24 @@ $DEPLOY_CMD -Durl=$SERVER_URL -DrepositoryId=$SERVER_ID \
 # Distribution jar is a shaded artifact so use the reduced dependency pom.
 $DEPLOY_CMD -Durl=$SERVER_URL -DrepositoryId=$SERVER_ID \
             $SRC_DOC_JARS \
-            -Dfile=$FPATH.jar -DpomFile=$AGGREGATOR_PL/dependency-reduced-pom.xml
+            -Dfile=$FPATH.jar -DgroupId=com.nvidia -DartifactId=$ART_ID -Dversion=$ART_VER
```

Comment:
The `artifactId` of the `aggregator/dependency-reduced-pom.xml` is `rapids-4-spark-aggregator_2.12`, we need to change it to `rapids-4-spark_2.12`.
https://github.com/NVIDIA/spark-rapids/blob/branch-21.10/aggregator/pom.xml#L27

Otherwise it will upload rapids-4-spark-aggregator_2.12 jars to OSS repo

```
[2021-09-26T02:07:25.867Z] [INFO] Metadata[project com.nvidia:rapids-4-spark-aggregator_2.12].filename = rapids-4-spark-aggregator_2.12-21.10.0.pom
[2021-09-26T02:07:25.867Z] [INFO] Metadata[gpg signature com.nvidia:rapids-4-spark-aggregator_2.12:jar:null:pom].filename = rapids-4-spark-aggregator_2.12-21.10.0.pom.asc
[2021-09-26T02:07:26.122Z] [INFO] Uploading: https://oss.sonatype.org/service/local/staging/deploy/maven2/com/nvidia/rapids-4-spark-aggregator_2.12/21.10.0/rapids-4-spark-aggregator_2.12-21.10.0-sources.jar
[2021-09-26T02:07:26.680Z] [INFO] Uploaded: https://oss.sonatype.org/service/local/staging/deploy/maven2/com/nvidia/rapids-4-spark-aggregator_2.12/21.10.0/rapids-4-spark-aggregator_2.12-21.10.0-sources.jar (802 KB at 1079.1 KB/sec)
[2021-09-26T02:07:26.680Z] [INFO] Downloading: https://oss.sonatype.org/service/local/staging/deploy/maven2/com/nvidia/rapids-4-spark-aggregator_2.12/maven-metadata.xml
[2021-09-26T02:07:26.934Z] [INFO] Downloaded: https://oss.sonatype.org/service/local/staging/deploy/maven2/com/nvidia/rapids-4-spark-aggregator_2.12/maven-metadata.xml (324 B at 1.4 KB/sec)
[2021-09-26T02:07:26.934Z] [INFO] Uploading: https://oss.sonatype.org/service/local/staging/deploy/maven2/com/nvidia/rapids-4-spark-aggregator_2.12/maven-metadata.xml
[2021-09-26T02:07:27.492Z] [INFO] Uploaded: https://oss.sonatype.org/service/local/staging/deploy/maven2/com/nvidia/rapids-4-spark-aggregator_2.12/maven-metadata.xml (324 B at 0.9 KB/sec)
[2021-09-26T02:07:27.492Z] [INFO] Uploading: https://oss.sonatype.org/service/local/staging/deploy/maven2/com/nvidia/rapids-4-spark-aggregator_2.12/21.10.0/rapids-4-spark-aggregator_2.12-21.10.0-sources.jar.asc
[2021-09-26T02:07:27.746Z] [INFO] Uploaded: https://oss.sonatype.org/service/local/staging/deploy/maven2/com/nvidia/rapids-4-spark-aggregator_2.12/21.10.0/rapids-4-spark-aggregator_2.12-21.10.0-sources.jar.asc (473 B at 1.3 KB/sec)
[2021-09-26T02:07:27.746Z] [INFO] Metadata[gpg signature com.nvidia:rapids-4-spark-aggregator_2.12:jar:sources].filename = rapids-4-spark-aggregator_2.12-21.10.0-sources.jar.asc
[2021-09-26T02:07:28.000Z] [INFO] Uploading: https://oss.sonatype.org/service/local/staging/deploy/maven2/com/nvidia/rapids-4-spark-aggregator_2.12/21.10.0/rapids-4-spark-aggregator_2.12-21.10.0-javadoc.jar
[2021-09-26T02:07:29.358Z] [INFO] Uploaded: https://oss.sonatype.org/service/local/staging/deploy/maven2/com/nvidia/rapids-4-spark-aggregator_2.12/21.10.0/rapids-4-spark-aggregator_2.12-21.10.0-javadoc.jar (18603 KB at 14822.8 KB/sec)
[2021-09-26T02:07:29.358Z] [INFO] Downloading: https://oss.sonatype.org/service/local/staging/deploy/maven2/com/nvidia/rapids-4-spark-aggregator_2.12/maven-metadata.xml
[2021-09-26T02:07:29.612Z] [INFO] Downloaded: https://oss.sonatype.org/service/local/staging/deploy/maven2/com/nvidia/rapids-4-spark-aggregator_2.12/maven-metadata.xml (324 B at 0.8 KB/sec)
[2021-09-26T02:07:29.612Z] [INFO] Uploading: https://oss.sonatype.org/service/local/staging/deploy/maven2/com/nvidia/rapids-4-spark-aggregator_2.12/maven-metadata.xml
[2021-09-26T02:07:29.867Z] [INFO] Uploaded: https://oss.sonatype.org/service/local/staging/deploy/maven2/com/nvidia/rapids-4-spark-aggregator_2.12/maven-metadata.xml (324 B at 0.9 KB/sec)
[2021-09-26T02:07:29.867Z] [INFO] Uploading: https://oss.sonatype.org/service/local/staging/deploy/maven2/com/nvidia/rapids-4-spark-aggregator_2.12/21.10.0/rapids-4-spark-aggregator_2.12-21.10.0-javadoc.jar.asc
```